.github/workflows/demos.yml

```diff
@@ -84,6 +84,11 @@ jobs:
           pkg_path = dirname(Base.find_package("RegularizedOptimization"))
           include(joinpath(pkg_path, "..", "examples", "demo-nnmf-constr.jl"))
         shell: julia --project="examples" --color=yes {0}
+      - name: Run SVM demo
+        run: |
+          pkg_path = dirname(Base.find_package("RegularizedOptimization"))
+          include(joinpath(pkg_path, "..", "examples", "demo-svm.jl"))
+        shell: julia --project="examples" --color=yes {0}
```

Comment:
Is the step "Installing non registered dependencies" still necessary here? Could it be the source of the problem?